platform/default/src/mbgl/storage/default_file_source.cpp

```diff
@@ -82,9 +82,9 @@ class DefaultFileSource::Impl {
         }
     }
 
-    void deleteRegion(OfflineRegion&& region, std::function<void (std::exception_ptr)> callback) {
+    void deleteRegion(OfflineRegion&& region, bool skipPack, std::function<void(std::exception_ptr)> callback) {
```

Comment:
I just don't like the fact this is a double negative.

`false` here means doing pack. What about just `doPack` or `pack`?